staffjoy/resources/schedule_timeclock.py

```diff
@@ -0,0 +1,5 @@
+from ..resource import Resource
+
+
+class ScheduleTimeclock(Resource):
+    PATH = "organizations/{organization_id}/locations/{location_id}/roles/{role_id}/schedules/{schedule_id}/timeclocks/"
```

Comment:
this is a read collection only endpoint to get timeclocks that occur during a schedule